raiden/tests/unit/test_tokennetwork.py

```diff
@@ -1126,5 +1141,290 @@ def test_routing_priority(
         to_address=address3,
         amount=1,
         previous_address=None,
+        config={},
     )
     assert routes[0].node_address == address2
+
+
+def test_routing_mocked_pfs(
```

Comment:
Normally I'd say it would be cleaner to split this into multiple tests (e.g. success, fail, no_route), but I guess to did it this way to reduce overhead with re-creating the topology multiple times?

Reply:
Yes, the setup is a bit involved, but I can move it into a fixture if you prefer.

Reply:
Done, please have another look.